pytest-embedded-idf/tests/test_idf.py

```diff
@@ -716,6 +716,90 @@ def test_dut_run_all_single_board_cases(dut):
     assert multi_stage.attrib['name'] == 'multiple_stages_test'
 
 
+def test_dut_run_all_single_board_cases_group(testdir):
+    testdir.makepyfile(r"""
+        def test_dut_run_all_single_board_cases(dut):
+            dut.run_all_single_board_cases(group="normal_case", timeout=10)
+    """)
+    testdir.runpytest(
+        '-s',
+        '--embedded-services', 'esp,idf',
+        '--app-path', os.path.join(testdir.tmpdir, 'unit_test_app_esp32'),
+        '--log-cli-level', 'DEBUG',
```

Comment:
you're not checking the debug level in the test case, maybe remove the log cli level here?

Reply:
yes, sure